tests/integration/components/policy-card-test.js

```diff
@@ -1,32 +1,39 @@
-/* eslint-disable ember/no-get */
-import { A } from '@ember/array';
-import EmberObject, { get } from '@ember/object';
 import { setupRenderingTest } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
 import { module, test } from 'qunit';
 import { render, click } from '@ember/test-helpers';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
 
 module('Integration | Component | policy card', (hooks) => {
   setupRenderingTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(function () {
+    this.store = this.owner.lookup('service:store');
+  });
 
   test('it renders when given expected data', async function (assert) {
-    const policy = EmberObject.create({
-      repositories: A(),
-      description: 'This is a moo-scription',
-      title: 'Moo title',
-      id: 1,
+    const submissionAttrs = this.server.create('submission').attrs;
```

Comment:
Pretty much all of the changes in this file involve moving to mirage and factories for the mock data and then also using the new data test selectors to be more declarative about what is being tested.